recipes/picobench/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+
+conan_basic_setup(TARGETS)
+
+find_package(picobench REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} ../test_package/test_package.cpp)
+target_link_libraries(${PROJECT_NAME} picobench::picobench)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

```suggestion
conan_basic_setup(TARGETS)

add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package/)
```
Re-use cmake file from test_package